cli/flox/src/commands/environment.rs

```diff
@@ -137,7 +137,15 @@ impl Edit {
         // decides to stop.
         loop {
             let new_manifest = Edit::edited_manifest_contents(&tmp_manifest, &editor)?;
-            match environment.edit(&flox, new_manifest).await {
+
+            let result = Dialog {
+                message: "Validating environment...",
```

Comment:
nit, I think it would be more consistent to have:
```suggestion
                message: "Building environment...",
```

Reply:
More consistent but more accurate for in the context I think

Reply:
What about `Building environment to validate edit..`? `Validating` by itself sounds like we're running tests, which isn't the case.

Reply:
Yeah that sounds reasonable